empty/database/knex.js

```diff
@@ -0,0 +1,12 @@
+
+ var knex = require('knex') ({
+  client: "pg",
+  connection: {
+    host: 'localhost',
```

Comment:
If you are going to be deploying this, you will need this to be a `process.env` variable as well.